File: VB.NET/VBBankingApp/App_Code/Utilities.vb
Imports Microsoft.VisualBasic

Public Module Utilities
    Public Function SingleQuote(ByVal str As String) As String
        Return "'" + str + "'"

    End Function


    Public Sub MessageBox(ByRef page As System.Web.UI.Page, ByVal message As String)

        Dim strScript As String = "<script language=JavaScript>"
        strScript += "alert('" + message + "');"
        strScript += "</script>"

        If (page.ClientScript.IsStartupScriptRegistered("messageBox") = False) Then
            page.ClientScript.RegisterClientScriptBlock(page.GetType(), "messageBox", strScript)
        End If
    End Sub

    Public Function EscapeSingleQuote(ByVal value As String) As String

        'Replace each single quote sql literal with two single quotes specific to MS-SQL
        Return "'" + value.Replace("'", "''") + "'"
    End Function

End Module

File: CSharp.NET/CSharpBankingApp/App_Code/Utilities.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;

/// <summary>
/// Summary description for Utilities
/// </summary>
public class Utilities
{
	public Utilities()
	{
		//
		// TODO: Add constructor logic here
		//
	}

    public static string SingleQuote(string value)
    {
        return "'" + value + "'";
    }


    public static void MessageBox(System.Web.UI.Page page, string message)
    {
        string strScript = "<script language=JavaScript>";
        strScript += "alert('" + message + "');";
        strScript += "</script>";

        if (page.ClientScript.IsStartupScriptRegistered("messageBox") == false)
        {
            page.ClientScript.RegisterClientScriptBlock(page.GetType(), "messageBox", strScript);
        }
    }

    public static string EscapeSingleQuote(string value)
    {
        //Replace each single quote sql literal with two single quotes
        return "'" + value.Replace("'", "''") + "'";
    }
}
